### ISSUE TITLE
Don't warn about react/jsx-no-target-blank when referrerPolicy is used

### ISSUE THREAD

**Issue by marusak** [yannickcr/eslint-plugin-react#2398]
We have in our code:
```
<a rel="noopener" referrerPolicy="no-referrer" target="_blank" href={fooURL}>foo</a>
```
But `react/jsx-no-target-blank` is complaining about it with:
```
 Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank
```

Isn't it the same to have `rel="noopener"` and `referrerPolicy="no-referrer"`  as having `rel="noopener noreferrer"`?

`eslint-plugin-react@7.14.3`

**Comment by ljharb** [comment]
What is “referrerPolicy”?

**Comment by marusak** [comment]
https://developer.mozilla.org/en-US/docs/Web/API/HTMLAnchorElement/referrerPolicy
or https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element

**Comment by ljharb** [comment]
Given that a number of browsers don't support it, it doesn’t seem safe to rely on.

**Comment by ianjmacintosh** [comment]
This is a pretty old issue (apologies for the thread necromancy), but I see this is now supported across all modern browsers: https://developer.mozilla.org/en-US/docs/Web/API/HTMLAnchorElement/referrerPolicy#browser_compatibility

**Comment by ljharb** [comment]
Many sites still need to support IE 9, let alone IE 11, but that's certainly a better list than 1.5 years ago.

It seems reasonable to allow for relying on this via an option.

**Comment by nickmccurdy** [comment]
IE is no longer supported, so we should be able to enable this by default.

**Comment by ljharb** [comment]
Supported is irrelevant if it's still used; I remain willing to add an option for it, and we can discuss changing the default in the future.
